src/content/cloud/registry.mdx

```diff
@@ -50,3 +50,25 @@ Any image pushed into the Okteto Registry is private. You'll need to authenticat
 Namespaces in Okteto Cloud are automatically allowed to pull images that belong to their namespace automatically. If your application uses images from the Okteto Registry, it'll be able to pull container images without any extra configuration.
 
 To pull an image from a different namespace, you'll need to create and configure the required `imagePullSecrets` as outlined [here](/docs/reference/faqs/#how-to-use-private-images).
+
+## Push helm chart into the Okteto Registry
+
+Just as we can push images to the Okteto Registry, you also have the option to push helm charts. To do this, the first step is to login to the registry where we want to push the chart:
+
+```
+helm registry login -u <user> <registry-domain>
+```
+
+Once you are logged against the Okteto Registry and packaged the chart, you are able to push the chart to the Okteto Registry.
+
+```
+helm push <packaged-chart> oci://<registry-domain>/<okteto-namespace>
```

Comment:
can use a specific URL for the example, like we do for the registry (and probably add a note saying that they should change the URL to match their instance's URL